src/Discord.Net.Rest/Entities/AuditLogs/DataTypes/ScheduledEventUpdateAuditLogData.cs

```diff
@@ -35,13 +36,13 @@ internal static ScheduledEventUpdateAuditLogData Create(BaseDiscordClient discor
             var status = entry.Changes.FirstOrDefault(x => x.ChangedProperty == "status");
             var entityType = entry.Changes.FirstOrDefault(x => x.ChangedProperty == "entity_type");
             var entityId = entry.Changes.FirstOrDefault(x => x.ChangedProperty == "entity_id");
-            var entityMetadata = entry.Changes.FirstOrDefault(x => x.ChangedProperty == "entity_metadata");
+            var location = entry.Changes.FirstOrDefault(x => x.ChangedProperty == "location");
             var userCount = entry.Changes.FirstOrDefault(x => x.ChangedProperty == "user_count");
             var image = entry.Changes.FirstOrDefault(x => x.ChangedProperty == "image");
 
             var before = new ScheduledEventInfo(
                 guildId?.OldValue.ToObject<ulong>(discord.ApiClient.Serializer),
-                channelId?.OldValue.ToObject<ulong?>(discord.ApiClient.Serializer),
+                channelId == null ? null : channelId.OldValue?.ToObject<ulong?>(discord.ApiClient.Serializer) ?? 0,
```

Comment:
Ugly expression, should be similarly written to the code below

Reply:
you literally cannot convert this in a null coalescing expression because of bracket and calling rules